### ISSUE TITLE
Proposal to add changeable username conditions

### ISSUE THREAD

**Issue by libneko** [ppy/osu-web#6451]
Username changes is blocked by reason of ranked beatmap or badges (or something unknown), even though there's no users who have the username which I was using.

![Username Change](https://sylphi.s-ul.eu/JBChk9yl.png "Username Change")
![Profile](https://sylphi.s-ul.eu/la9GjDHp.png "Profile")
![Username Search](https://sylphi.s-ul.eu/d9VmXM2E.png "Username Search")

Excepting some cases, such of these username changes do not affect in critical way. so It might be fine to allow changing username to own previous username with some conditions, in my opinion.

Conditions using the previous username (Example):
- There's no ranked/approved beatmaps which is not mine for the previous username.
- Someone has the previous username, but he is inactivity a long time and doesn't have ranked/approved beatmaps.

**Comment by gagahpangeran** [comment]
Could you just request to support to revert your name for this case?

**Comment by libneko** [comment]
I already tried, and I received a mail to use username changes in osu!store.

**Comment by gagahpangeran** [comment]
After I check old issue, looks like duplicate of #1881

**Comment by peppy** [comment]
Closing as duplicate. Please contact support for now, as we can manually fix such cases.
